packages/browser/src/plugins/ajs-destination/utils.ts

```diff
@@ -0,0 +1,34 @@
+import { Integrations } from '@segment/analytics-core'
+import { LegacyIntegrationConfiguration } from '../..'
+
+export const isInstallableIntegration = (
+  name: string,
+  integrationSettings: LegacyIntegrationConfiguration
+) => {
+  const { type, bundlingStatus, versionSettings } = integrationSettings
+  // We use `!== 'unbundled'` (versus `=== 'bundled'`) to be inclusive of
+  // destinations without a defined value for `bundlingStatus`
+  const deviceMode =
+    bundlingStatus !== 'unbundled' &&
+    (type === 'browser' || versionSettings?.componentTypes?.includes('browser'))
+
+  // checking for iterable is a quick fix we need in place to prevent
+  // errors showing Iterable as a failed destiantion. Ideally, we should
+  // fix the Iterable metadata instead, but that's a longer process.
+  return (deviceMode || name === 'Segment.io') && name !== 'Iterable'
+}
+
+export const isDisabledIntegration = (
+  integrationName: string,
+  globalIntegrations: Integrations
+) => {
+  const allDisableAndNotDefined =
+    globalIntegrations.All === false &&
+    globalIntegrations[integrationName] === undefined
+
+  return (
+    integrationName.startsWith('Segment') ||
```

Comment:
https://github.com/segmentio/analytics-next/pull/669/files/316f28152ddd76eab30d522c80d9cede525fc336#r1035384414

Reply:
https://github.com/segmentio/analytics-next/pull/669#discussion_r1035384414

Reply:
To me, it makes more sense to have the `Segment` check in `isInstallableIntegrations` since `Segment` is not a classic integration. This function seems to imply you're simply disabling integrations.

True the end result is the same but makes `isInstallableIntegrations` more truthy.